### PR TITLE
Fix non aws deployment

### PR DESCRIPTION
**Pull request by Szpadel**

### CODE REVIEW

File: roles/cs.aws-magento-cron/tasks/main.yml [hunk 1]
```diff
@@ -27,6 +27,7 @@
         name: magento-cron-aws
         state: stopped
         enabled: no
+      ignore_errors: yes
```
Comment (krisdante) [inline roles/cs.aws-magento-cron/tasks/main.yml:30]: Ugly. Can't we make it nicer?
Why this is even executed for non-aws?

File: roles/cs.nginx-https-termination/templates/vhost.conf.j2 [hunk 2]
```diff
@@ -36,12 +36,10 @@ server {
     listen 80;
 {% endif %}
 
-    listen 443 ssl http2;
+    {{ ssl_config(vhost) }}
```
Comment (krisdante) [inline roles/cs.nginx-https-termination/templates/vhost.conf.j2:39]: @pinkeen Haven't you been adding something similar to master already?

Comment (pinkeen) [reply roles/cs.nginx-https-termination/templates/vhost.conf.j2:39]: This is strange, this commit is already on `develop` branch. I don't know how it got into this MR

File: roles/cs.aws-magento-cron/tasks/main.yml (hunk 1 above)
Comment (pinkeen) [reply roles/cs.aws-magento-cron/tasks/main.yml:30]: Shouldn't this *whole role* be skipped when not on AWS?

PS `ignore_errors` is *evil* - it should be used only if you're executing a command that you *expect* to fail in some cases and then compare the registered _result_ to get some information.

**Pull request merged**